### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by DaPandaMan** [KelvinTegelaar/AutotaskAPI#27]
The "Contacts" resource is not valid for New-AutoTaskAPIResource or Set-AutoTaskAPIResource

If you attempt to use "Contacts" as the resource parameter for either command, it returns a server 404 error. 
If you attempt to use "Contacts" as the resource parameter for New-AutoTaskBody, it will respond with Valid parameters, and Contacts is not listed. 

 - OS: Windows 10 
 - PowerShell version: V5.1, 

"Contacts" does work as a valid resource parameter for Get-AutotaskAPIResource.
From what I can tell, I don't think the POST information is listed correctly in the V1.json

Thanks!

**Comment by KelvinTegelaar** [comment]
Yup! Requested an update at Autotask for this. ETA is the next minor release.
